src/main/java/dev/jbang/Script.java

```diff
@@ -658,4 +659,12 @@ public String getJavaAgentOption() {
 		return agentOptions;
 	}
 
+	public File getArgs() {
```

Comment:
this seems unnecessary to store on the script ?

Reply:
Removed.
It was because I was not limiting by length. So I needed for the test.